internal/service/kafka/kafka_topic_cache.go

```diff
@@ -136,3 +136,13 @@ func (t *kafkaTopicCache) GetQueue(projectName, serviceName string) []string {
 
 	return t.inQueue[projectName+serviceName]
 }
+
+// FlushTopicCache for tests only!
+func FlushTopicCache() {
```

Comment:
could this be moved to the test package itself? iirc, test packages also have access to non-exported functions